### PR TITLE
Update email copy for the census form submission

### PR DESCRIPTION
**Pull request by Erin007**
Folks will now receive an email with this [copy ](https://docs.google.com/document/d/1S_i85fFlJXZwPdWrJWNdyP1EzUxzf6QUBmZzZms6dkI/edit) when they submit the census form on /yourschool.

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: pegasus/emails/census_form_receipt.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+from: 'Hadi Partovi (Code.org) <hadi_partovi@code.org>'
+subject: "Your local school’s computer science program"
 ---
-Hi,
+Thank you for sharing information about your school’s computer science program and your continued support of our mission to give every student the opportunity to learn computer science.
 
-Thank you for submitting information about your school. Marina will decide on some more text to go here.
+Here are steps that you can take to expand computer science access to more students in your local region:
+If you’re a district or school administrator, see our [Administrator guide](https://code.org/administrators) for expanding access in your school or district.
+If you’re a teacher, view our curriculum options for [teaching computer science](https://code.org/teach) to your students - no computer science experience necessary
```
Comment (breville) [inline pegasus/emails/census_form_receipt.md]: Missing period?

File: pegasus/emails/census_form_receipt.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+If you’re a parent or community member, [take an action](https://code.org/help) to encourage your local school to offer computer science
```
Comment (breville) [inline pegasus/emails/census_form_receipt.md]: Missing period?

File: pegasus/emails/census_form_receipt.md [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
-Thank you for submitting information about your school. Marina will decide on some more text to go here.
+Here are steps that you can take to expand computer science access to more students in your local region:
+If you’re a district or school administrator, see our [Administrator guide](https://code.org/administrators) for expanding access in your school or district.
+If you’re a teacher, view our curriculum options for [teaching computer science](https://code.org/teach) to your students - no computer science experience necessary.
+If you’re a parent or community member, [take an action](https://code.org/help) to encourage your local school to offer computer science.
 
+In just a few years, support for computer science has impacted hundreds of thousands of classrooms and tens of millions of students. Many school districts, U.S. states, and even entire countries have joined the movement.
+
+Please follow our progress on [Facebook](https://www.facebook.com/Code.org) or on [Twitter](https://twitter.com/codeorg) to stay in touch.
+Thank you so much for your support,
```
Comment (tanyaparker) [inline pegasus/emails/census_form_receipt.md]: I'm not sure how this looks, but `Thank you so much for your support,` should be on a new line IMO

Comment (tanyaparker) [reply pegasus/emails/census_form_receipt.md]: With a new line/space between `touch.` and `Thank you`

**Pull request merged**